client/src/app/site/agenda/components/list-of-speakers/list-of-speakers.component.html

```diff
@@ -91,7 +91,7 @@ <h1 class="los-title">
 
     <!-- Waiting speakers -->
     <div class="waiting-list" *ngIf="speakers && speakers.length > 0">
-        <os-sorting-list [input]="speakers" [live]="!isSortMode" [count]="true" [enable]="opCanManage() && isSortMode">
+        <os-sorting-list [input]="speakers" [live]="!isMobile" [count]="true" [enable]="opCanManage() && isSortMode">
```

Comment:
Apparently, it's not yet working.
"live" Sorting does not send a request, thus nor really updating the list of speakers.
Reloading or having a second tab shows that sorting has no effect